i2c/oled_i2c/img_to_array.py

```diff
@@ -26,22 +26,20 @@
 try:
     im = Image.open(img_path)
 except OSError:
-    print("Oops! The image could not be opened.")
-    sys.exit()
+    raise Exception("Oops! The image could not be opened.")
 
 if im.size[0] > OLED_WIDTH or im.size[1] > OLED_HEIGHT:
-    print("OLED display only 128 pixels wide and 32 pixels high!")
-    sys.exit()
+    raise Exception(f"OLED display only {OLED_WIDTH} pixels wide and {OLED_HEIGHT} pixels high!")
```

Comment:
Perhaps this should also print out the dimensions of the image, to make it clearer what the problem actually is? :wink: